src/component/tag/tag.scss

```diff
@@ -1,26 +1,17 @@
 @import '@/style/fonts.scss';
 @import '@/style/theme.scss';
 
-$colors: (
-  'primary-color': 'background-primary-variant',
-  'primary-color-variant-3': 'background-primary-variant-3',
-  'primary-color-variant-4': 'background-primary-variant-4'
-);
-
 .okp4-dataverse-portal-tag-main {
-  @include with-theme {
-    @include noto-sans(700);
-    border-radius: 0 20px;
-    padding: 8px;
-    font-size: 14px;
-    line-height: 24px;
-    letter-spacing: 2px;
-
-    @each $color-key, $color-value in $colors {
-      &.#{$color-key} {
-        color: themed($color-key);
-        background-color: themed($color-value);
-      }
-    }
-  }
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $primary-variant-3;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 22px;
+  font-family: 'Noto Sans';
+  height: 34px;
+  padding: 0 8px;
```

Comment:
Missing padding top and bottom

Reply:
We would've been more explicit by writing `padding: 0 8px 0 8px`. The `padding` shorthand property can be simplified when both top/bottom or left/right are similar. In that case `padding: 0 8px` is equivalent to `padding: 0 8px 0 8px`

Reply:
I used fix height so we dont need to add padding top and bot.
If the problem is the case of two lines, we can block a number of characters and use "..." at the end in CSS.
But normally we don't want a tags in two lines :)

Reply:
Fair point 👍